src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -34,7 +34,7 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
   }
 
   const { body } = req;
-  const { reviewerId, crId, reviewNotes, accepted } = body;
+  const { reviewerId, crId, reviewNotes, accepted, psId } = body;
```

Comment:
you have to update change-requests.routes.ts to include this as an optional parameter in the post